keras_cv/ops/__init__.py

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from keras_cv.keypoint.converters import convert_format
-from keras_cv.keypoint.formats import REL_XY
-from keras_cv.keypoint.formats import XY
-from keras_cv.keypoint.utils import filter_out_of_image
+from keras_cv.ops.keypoint_converters import convert_keypoint_format
+from keras_cv.ops.keypoint_formats import REL_XY
+from keras_cv.ops.keypoint_formats import XY
+from keras_cv.ops.keypoint_utils import filter_out_of_image
```

Comment:
Should we have keypoints in the filter_out_of_image?

Reply:
`keras_cv.ops.filter_out_of_image` is bit out of the context of keypoint (assuming its keypoints specific.)

Also `out_of` is bit weird as well, how about `filter_image`?

Reply:
Done.
filter_keypoints is more natural (so we can do filter_boxes if we need)